extensions/README.md

```diff
@@ -0,0 +1 @@
+This is a folder for useful extension to be added for developer process

```

Comment:
Is this convention? Is it possible to add react dev tools as `devDependency` instead of checking into repo?

Reply:
I would like to that ideally [but it is more path based](https://electronjs.org/docs/tutorial/devtools-extension)

I felt it would keep the setup for each person for this simpler, the other way is through env variables